### DISCUSSION TITLE
求助！如何支持我这样的 monorepo 项目？

### DISCUSSION THREAD

**Discussion by jo0ger** [umijs/dumi#2046] [Q&A]
我的项目的目录结构大致如下（因为是不同组的人维护，所以每个组件就是个 package，并且各个组件内也含有 docs）：
```bash
├── docs
│   ├── guide
│   ├   |── intro.md
│   └── index.md
├── packages
│   ├── components
|   |   |── A
|   |   |   |── docs
|   |   |   |   |── index.md
|   |   |   |   |   |── cases
|   |   |   |   |   |   |── case1.md
|   |   |   |   |   |   |── case2.md
|   |   |── B
|   |   |   |── docs
|   |   |   |   |── index.md
|   |   |   |   |   |── cases
|   |   |   |   |   |   |── case1.md
|   |   |   |   |   |   |── case2.md
│   ├── plugins
|   |   |── C
|   |   |   |── docs
|   |   |   |   |── index.md
|   |   |   |   |   |── cases
|   |   |   |   |   |   |── case1.md
|   |   |   |   |   |   |── case2.md


```

页面的布局大致是：

顶部 nav：指南，组件，插件
侧边栏 sidebar：
 - nav 为组件时，侧边显示两个分组，A 和 B，A 和 B 下面分别有三个菜单，对应各自的 docs/index.md、docs/cases/case1.md、docs/cases/case2.md
 - nav 为插件时，和上面类似

sidebar 想达到这样的效果
![image](https://github.com/umijs/dumi/assets/16385416/f0286c31-0f71-45c4-8833-100cd2d6dce7)

这种情况下，.dumirc.ts 里的 `resolve.atomDirs` 怎么配置？配置了两天了，始终无法在 A/B group 下，放 3 个菜单

**Comment by yuli-xtransfer** [comment]
sidebar 的配置里一二级菜单应该支持的吧
